app/src/main/java/com/elmeradrianv/shesafe/MainActivity.java

```diff
@@ -23,20 +24,20 @@ protected void onCreate(Bundle savedInstanceState) {
         final FragmentManager fragmentManager = getSupportFragmentManager();
         // define your fragments here
         final Fragment mapFragment = new MapFragment();
-//        final Fragment fragment2 = new SecondFragment();
+        final Fragment tableFragment = new TableViewFragment();
 //        final Fragment fragment3 = new ThirdFragment();
 
-        setupBottomNavigationView(bottomNavigationView, fragmentManager,mapFragment);
+        setupBottomNavigationView(bottomNavigationView, fragmentManager, mapFragment,tableFragment);
 
     }
 
-    private void setupBottomNavigationView(BottomNavigationView bottomNavigationView, FragmentManager fragmentManager, Fragment mapFragment) {
+    private void setupBottomNavigationView(BottomNavigationView bottomNavigationView, FragmentManager fragmentManager, Fragment mapFragment, Fragment tableFragment) {
```

Comment:
I would refactor this to not have any variables passed in. So just:

setupBottomNavigationView()

And then you can "inline" most of the calls. So instead of something like:

fragment = mapFragment;
it can just be
fragment = new MapFragment()